build.gradle

```diff
@@ -41,7 +41,7 @@ dependencies {
         implementation 'com.epam.reportportal:client-java:5.0.0-BETA-17'
         implementation 'com.epam.reportportal:commons-model:5.0.0'
     } else {
-        implementation 'com.epam.reportportal:client-java:5.0.0-BETA-17'
+        implementation 'com.github.reportportal:client-java:71f257b'
```

Comment:
Please merge client-java PR first

Reply:
It also requires a reference in a README.md file that we publish anonymous statistics, which cannot be turned off. Please ask Katsia for details.

Reply:
updated